rules/cloud/azure/identity_protection/azure_identity_protectection_anomalous_token.yml

```
@@ -0,0 +1,22 @@
+title: Anomalous Token
```

Can you elaborate more? For example say something like `Potential Azure AD Anomalous Token`
```suggestion
title: Potential Azure AD Anomalous Token
```

---

"Potential" is a keyword used to indicate level of confidence to the reader of the alert. Saying `Anomalous Token` is a guarantee and it'll create confusion if the alert triggers and its not. Hence why its best to clear it in the title as it's the first thing the user will see.

As for the Product the whole folder is called `Azure` so will need to change that. And since rule could be ingested by people with multiple cloud techno its best to specify things as clear as possible in the title and description as they're the indicators for the analyst.

---

It's important to remember that not everyone is an expert in every field.
I may be Paladins level 90 for the on premise, but I'm level 1 on detection in the cloud. 😄 

So the title and description must provide the best possible orientation for the operator.

---

I still disagree. Every alert that fires is POTENTIALLY a false positive. Do we have 'potential' in front of every alert then? Please link me to ones that we feel are 100% accurate that don't have an false positives and show me another one that has 'potential' in front of it. Otherwise I don't understand why we would add that to this. The alert is firing because the system is seeing enough to give it indication that something is risky with the sign-in. It needs to be investigated by analyst. 

We have another alert type called "Impossible Travel". Sometimes a user has their username and password stolen and someone logs in with halfway across the globe and the alert fires. Sometimes the user logs in with a VPN does some stuff, then unplugs goes to the coffee shop and does some stuff and forgets to login to the VPN. The alert fires. What title would you call that? 

Ultimately I will go with whatever you all want for all these titles. This is your project, we are just trying to help contribute and make sure folks are investigating the things they should be investigating.

---

I think we all agree that "Anomalous Token" is too imprecise as a title.
A `high` alert should be considered as valid by the operator.
When I read the documentation:
"Because this is a high noise detection, there's a higher than normal chance that some of the sessions flagged by this detection are false positives"
So for me it  should be at best medium.
The documentation uses potential "the tenant admin should consider this risk as an indicator of potential token replay".

@nasbench perhaps we should "define" a vocabulary for titles according to level. More like an aid.